src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/CodeActions/CodeActionEndToEndTest.NetFx.cs

```diff
@@ -1306,4 +1409,78 @@ static IEnumerable<TagHelperDescriptor> BuildTagHelpers()
             }
         }
     }
+
+    private class ExtractToComponentResolverDocumentContextFactory : TestDocumentContextFactory
```

Comment:
This file is getting fairly large. Can we make it either a partial class and create `CodeActionEndToEndTest.ExtractToComponent.NetFx` or make a base class for these?